From 56d6656f13e223e94ad51b1168e26d578ebca249 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Wed, 24 Jun 2015 20:47:24 +0800
Subject: [PATCH] dts: rk3368-tb_mipi: disable headset wakeup.

because the hp det src is provider by micbais in this board.
when suspend, micbais will be power down to reduce power consumption.

Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/boot/dts/rk3368-tb_mipi.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rk3368-tb_mipi.dts b/arch/arm64/boot/dts/rk3368-tb_mipi.dts
index 66274cd6fdd4..fa7275a91566 100644
--- a/arch/arm64/boot/dts/rk3368-tb_mipi.dts
+++ b/arch/arm64/boot/dts/rk3368-tb_mipi.dts
@@ -611,6 +611,7 @@
 		pinctrl-names = "default";
 		pinctrl-0 = <&gpio0_c7>;//gpio0_c7
 		io-channels = <&adc 2>;
+		rockchip,headset_wakeup = <0>;
        /*
                hook_gpio = ;
                hook_down_type = ; //interrupt hook key down status
-- 
2.35.3

